test_scripts/AppServices/RPCPassThrough/009_UnknownRPC_Consumer_UNSUPPORTED_REQUEST.lua

```diff
@@ -0,0 +1,126 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
Perhaps rename the file to indicate that `allow_unknown_rpc_passthrough` had been set to false. Maybe something like `009_UnknownRPC_Passthrough_Disallowed`